TomTom SDK Examples/Examples/UseCase/Offline/Utils/OfflineConfig.swift

```diff
@@ -0,0 +1,29 @@
+//  © 2023 TomTom NV. All rights reserved.
+//
+//  This software is the proprietary copyright of TomTom NV and its subsidiaries and may be
+//  used for internal evaluation purposes or commercial use strictly subject to separate
+//  license agreement between you and TomTom NV. If you are the licensee, you are only permitted
+//  to use this software in accordance with the terms of your license agreement. If you are
+//  not the licensee, you are not authorized to use this software in any manner and should
+//  immediately return or destroy it.
+
+import Foundation
+
+enum OfflineConfig {
+    // Update Server
+    static let ndsUpdateServer = "https://api.tomtom.com/nds-test/updates/1/fetch"
```

Comment:
is it a final URL that should used?

Reply:
While using a final is not necessary, it is intended to be utilized within the configuration. Any suggestions or recommendations would be greatly appreciated.

Reply:
I see this hasn't got any other recommendations; solving it for now.